src/nvim/vim.h

```diff
@@ -27,9 +27,6 @@ Error: configure did not run properly.Check auto/config.log.
 # endif
 #endif
 
-/* user ID of root is usually zero, but not for everybody */
-#define ROOT_UID 0
```

Comment:
That's what everyone says when they use magic numbers. There's no good reason to remove this define--its existence costs nothing and adds some finite amount of clarity.